operator/v1/types_ingress.go

```diff
@@ -1030,6 +1035,22 @@ type IngressControllerHTTPHeaderBuffer struct {
 	// +kubebuilder:validation:Minimum=4096
 	// +optional
 	HeaderBufferMaxRewriteBytes int32 `json:"headerBufferMaxRewriteBytes,omitempty"`
+	// threadCount defines the number of threads created per HAProxy process.
```

Comment:
Could you put a blank line after line 1037? :grin:

Reply:
will do